actions.py

```diff
@@ -143,12 +143,12 @@ def validate_time(
     ) -> Dict[Text, Any]:
         """Validate time value."""
 
-        if value:
+        try:
```

Comment:
how come we changed that to a try/except btw?

Reply:
The try/except also handles the case where there is a `value` but it isn't formattable (e.g. if it was a date range) - We're not expecting that since it asks for a single payment date, but i did it once by mistake and found out it breaks

Reply:
alright, then let's except the specific two errors and not just a generic try/except :)

Reply:
I think it already does except `TypeError`?

Reply:
yes sorry, i am silly